src/goPath.ts

```diff
@@ -142,6 +142,19 @@ export function parseEnvFile(path: string): { [key: string]: string } {
 	}
 }
 
+export function parseEnvFiles(paths: string[]): { [key: string]: string }[] {
+	const envs: { [key: string]: any }[] = [];
```

Comment:
I think you can shorten this function with `return paths.map(path => parseEnvFile(path));`